shared/data_lake.py

```diff
@@ -11,19 +11,16 @@ def upload_to_data_lake(azure_credential, file_name, data_str):
     container_name = os.environ.get("DATALAKE_GEN_2_CONTAINER_NAME")
     directory_name = os.environ.get("DATALAKE_GEN_2_DIRECTORY_NAME")
 
-    unique_file_path = f"{file_name}"

```

Comment:
unique_file_path seems to be exact same as file_name so not sure why new variable is needed?

Reply:
Left over from an edit - good catch.